purchase_fiscal_position_update/models/purchase.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2011-2014 Julius Network Solutions SARL <contact@julius.fr>
+# @author Mathieu Vatel <mathieu _at_ julius.fr>
+# © 2014-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.onchange('fiscal_position_id')
+    def fiscal_position_change(self):
```

Comment:
@pedrobaeza Your code proposal is better. @angelmoya 's code was not right. If an inherited module want to change something, he should do it in the map_tax() method of account.fiscal.position, not here. And by the way, your proposal is 4 lines of code ; mine is exactly the same, so it's not simpler. It just seems crazy for me to call onchange_product_id() which itself calls _suggest_quantity() (which reads supplierinfo !) and _onchange_quantity() (which calls 3 other methods !)